module/zfs/zfs_ioctl.c

```diff
@@ -6232,6 +6235,7 @@ static const zfs_ioc_key_t zfs_keys_send_new[] = {
 	{"embedok",		DATA_TYPE_BOOLEAN,	ZK_OPTIONAL},
 	{"compressok",		DATA_TYPE_BOOLEAN,	ZK_OPTIONAL},
 	{"rawok",		DATA_TYPE_BOOLEAN,	ZK_OPTIONAL},
+	{"partialok",		DATA_TYPE_BOOLEAN,	ZK_OPTIONAL},
```

Comment:
Please update `tests/zfs-tests/cmd/libzfs_input_check/libzfs_input_check.c` accordingly so it can validate this new optional parameter.